tower_cli/cli/misc.py

```diff
@@ -204,3 +207,39 @@ def config(key=None, value=None, scope='user', global_=False, unset=False):
             UserWarning
             )
     click.echo('Configuration updated successfully.')
+
+
+@click.command()
+@click.argument('username', required=True)
+@click.option('--password', required=True, prompt=True, hide_input=True)
+@click.option('--scope', required=False, default='write',
+              type=click.Choice(['read', 'write']))
+@click.option('-v', '--verbose', default=None,
```

Comment:
I opted to just specify `-v` manually instead of inheriting _all_ global options, because some of them don't really make sense for this command.